File: Unidad9/ChatClean/app/domain/entitites/clsMensajeUsuario.ts
// clsDepartamento.ts
export class clsMensajeUsuario {
  private nombre: string;
  private mensaje: string;

  constructor(nombre: string, mensaje: string) {
    this.nombre = nombre;
    this.mensaje = mensaje;
  }

    get _nombre() { return this.nombre; }
    set _nombre(value: string) { this.nombre = value; }

    get _mensaje() { return this.mensaje; }
    set _mensaje(value: string) { this.mensaje = value; }
}

File: Unidad9/Ejercicio1/Ejercicio1/Domain/Entities/clsMensajeUsuario.cs
namespace Ejercicio1.Domain.Entities
{
    public class clsMensajeUsuario
    {
        public string nombre { get; set; } = string.Empty;
        public string mensaje { get; set; } = string.Empty;

        public clsMensajeUsuario() { }

        public clsMensajeUsuario(string nombre, string mensaje)
        {
            this.nombre = nombre;
            this.mensaje = mensaje;
        }
    }
}
